pubpub/pubpub
# feature: pub meta Details pane

This PR adds the pub meta Details pane, which shows author, citation, and publication date information. Here it is in its natural habitat:
![image](https://user-images.githubusercontent.com/2208769/58595511-e69ad200-823e-11e9-9161-654aa7959293.png)
The PR includes a number of changes to more distant code to enable this, including work on the citation generation code. I'll call that out in the inline comments.

_Test plan:_
- Visit a pub and open the Details pane by clicking the "..." button in the pub header
- Try updating the pub attributions and affiliations and ensure they are reflected here
- Ensure both the "copy" and "more" buttons beneath the citation work properly.
